app/auth/authorize_api_request.rb

```diff
@@ -0,0 +1,43 @@
+# 매API 요청의 Header에 포함된 Token 유효성을 검증하는 클래스
+class AuthorizeApiRequest
+  def initialize(headers = {})
+    @headers = headers
+  end
+
+  # 유효한 user record를 리턴한다
+  def call
+    {
+      user: user
+    }
+  end
+
+  private
+
+  attr_reader :headers
+
+  def user
+    # database에서 유저를 찾는다
+    @user ||= User.find(decoded_auth_token[:user_id]) if decoded_auth_token
+    # User가 존재하지 않을 경우 에러
+  rescue ActiveRecord::RecordNotFound => e
+    # 잘못된 토큰 정보
+    raise(
+      ExceptionHandler::InvalidToken,
+      ("#{Message.invalid_token} #{e.message}")
+    )
```

Comment:
```suggestion
    raise(ExceptionHandler::InvalidToken,("#{Message.invalid_token} #{e.message}"))
```
해당 raise는 한줄로 정렬되는 것이 더 잘 읽힐 것 같아요!

또한 ruby 특징으로 괄호 생략 문법을 지원하는데 괄호의 댑스가 깊어진다면 괄호를 벗겨내는 것도 가독성 향상에 도움을 줄 때가 있더라고요!
특정 룰이 정해져 있는 것은 아니라 컨벤션이라 할 수 없겠지만 상황에 따라 유동적으로 적용되면 좋을 것 같아요~
```ruby
raise ExceptionHandler::InvalidToken, "#{Message.invalid_token} #{e.message}"
```

[프로젝트에서 쓰이는 raise] - 모든 것이 이렇게 되어있진 않지만 보편적으로 이렇게 작성되어 있습니다!
```ruby
raise Exceptions::InvalidParam.new(I18n.t('apple_login.code.required'), field: 'code', reason: 'required') unless authorization_code
```

Reply:
가독성 면에서 더 좋다는 점 동의합니다!
제안주신대로 한번 수정해볼께요 :)